.azure-pipelines-canary.yml

```diff
@@ -24,7 +24,7 @@ stages:
       androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
       iosProvisioningProfileFile: $(InternalProvisioningProfile)
       iosCertificateFile: $(InternalCertificate)
-      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS'
+      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS.TestFlight'
       removeHyperlinksFromReleaseNotes: true
 
 - stage: AppCenter_Canary
```

Comment:
Aren't we missing the job to push the canary build to testflight?

Reply:
Especially since we would build canary with the internal testflight variable group

Reply:
you're right, I'll check that